lib/webpack.config.js

```diff
@@ -2,6 +2,7 @@ const webpack = require('webpack');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const TerserPlugin = require('terser-webpack-plugin');
 const CssMinimizerPlugin = require('css-minimizer-webpack-plugin');
+const NodePolyfillPlugin = require('node-polyfill-webpack-plugin');
```

Comment:
Why do we need the polyfill without changing the webpack version?

Reply:
To successfully build `plotly.js` which started using `webpack` instead of `browserify` from [v2.17.0](https://github.com/plotly/plotly.js/releases/tag/v2.17.0)